motr/idx.c

```diff
@@ -157,20 +156,16 @@ static int idx_op_init(struct m0_idx *idx, int opcode,
 	m0_op_idx_bob_init(oi);
 	m0_ast_rc_bob_init(&oi->oi_ar);
 
-#if defined(DTM0)
-	if (M0_IN(op->op_code, (M0_IC_PUT, M0_IC_DEL))) {
-		m0c = entity->en_realm->re_instance;
+	if (ENABLE_DTM0 && M0_IN(op->op_code, (M0_IC_PUT, M0_IC_DEL))) {
```

Comment:
Why do we need to substitute #ifndef here with runtime check? That makes the resulting binary more heavy with the branch of code that never happen for particular build.